### ISSUE TITLE
[Bug]: Can't select more than one ExportTo options

### ISSUE THREAD

**Issue by cibermanchego** [silverhack/monkey365#26]
**What happened?**  
When running monkey365, I try different ExportTo options and none work, only when a single option is specified works.

**How to reproduce it**  
1. What command are you running?
$param = @{
    Instance = 'Azure';
    Analysis = 'All';
    PromptBehavior = 'SelectAccount';
    subscriptions = '000000-00000000-0000';
    TenantID = '000000-00000000-0000';
    ExportTo = "CSV,JSON,CLIXML,PRINT,EXCEL,HTML";
}
$assets = Invoke-Monkey365 @param
2. See error
Invoke-Monkey365: Cannot validate argument on parameter 'ExportTo'. The argument "JCSV,JSON,CLIXML,PRINT,EXCEL,HTML" does not belong to the set "CSV,JSON,CLIXML,PRINT,EXCEL,HTML" specified by the ValidateSet attribute. Supply an argument that is in the set and then try the command again.
**Expected behavior**  
Get output in the formats specified

**From where are you running Monkey365?**  
Please, complete the following information:
 - Resource: Workstation 
 - OS: Windows 10
 - PowerShell Version [`$PsVersionTable`]: 7.2.6
 - Monkey365 Version:
 - Others:

**Comment by cibermanchego** [comment]
Thank you for the quick response. True, there is a typo on the text I pasted, but I tried with several combinations(with no typos) and only worked with a single option. 
I was using the example from the documentation:

![image](https://user-images.githubusercontent.com/6847856/197792463-2d17efc9-d583-49fc-bf72-a1d071c6c7b5.png)

After I got the errors I should have checked the help for Invoke-Monkey365, where it says that ExportTo parameter it's an array; my bad.
Still I was able to use the tool, and it's great, thank you!
